src/bubbles.js

```diff
@@ -76,6 +76,8 @@ export default class Bubbles extends React.Component {
 
     const bubblesE = bubbles
       .enter()
+      .append('a')
+      .attr('xlink:href', d => d.link)
```

Comment:
Don't you want the link to open in a new tab? 
Anyhow, I am merging this one.
Thanks a lot for this. :)